gbpservice/nfp/service_vendor_agents/vyos/agent/src/config_server/vpn_api_server.py

```diff
@@ -1,28 +1,34 @@
-#!/usr/bin/env python
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import array
+import copy
+import fcntl
 import logging
-import json
-import netifaces
-import netaddr
 import socket
-import fcntl
 import struct
-import array
-import time
-import ast
-import copy
 import subprocess
-import os
-from netaddr import IPNetwork, IPAddress
+import time
+
+from execformat.executor import session
+from netaddr import IPAddress
+from netaddr import IPNetwork
 from operations import configOpts
 from vyos_session import utils
-from netifaces import AF_INET, AF_INET6, AF_LINK, AF_PACKET, AF_BRIDGE
-#from vyos_session.configsession import ConfigSession as session
-from execformat.executor import session
 
 OP_SUCCESS = True
```

Comment:
These constants are used in other modules too. Better to keep it in some constants file and import that file.